labapi/src/main/java/com/microsoft/identity/internal/test/labapi/api/DeleteDeviceApi.java

```diff
@@ -104,26 +98,26 @@ public com.squareup.okhttp.Call apiDeleteDeviceDeleteCall(String upn, String dev
                 public com.squareup.okhttp.Response intercept(com.squareup.okhttp.Interceptor.Chain chain) throws IOException {
                     com.squareup.okhttp.Response originalResponse = chain.proceed(chain.request());
                     return originalResponse.newBuilder()
-                    .body(new ProgressResponseBody(originalResponse.body(), progressListener))
-                    .build();
+                            .body(new ProgressResponseBody(originalResponse.body(), progressListener))
+                            .build();
                 }
             });
         }
 
         String[] localVarAuthNames = new String[] {  };
-        return apiClient.buildCall(localVarPath, "POST", localVarQueryParams, localVarCollectionQueryParams, localVarPostBody, localVarHeaderParams, localVarFormParams, localVarAuthNames, progressRequestListener);
+        return apiClient.buildCall(localVarPath, "DELETE", localVarQueryParams, localVarCollectionQueryParams, localVarPostBody, localVarHeaderParams, localVarFormParams, localVarAuthNames, progressRequestListener);
     }
-    
+
     @SuppressWarnings("rawtypes")
     private com.squareup.okhttp.Call apiDeleteDeviceDeleteValidateBeforeCall(String upn, String deviceid, final ProgressResponseBody.ProgressListener progressListener, final ProgressRequestBody.ProgressRequestListener progressRequestListener) throws ApiException {
-        
+
         com.squareup.okhttp.Call call = apiDeleteDeviceDeleteCall(upn, deviceid, progressListener, progressRequestListener);
         return call;
 
-        
-        
-        
-        
+
+
+
+
```

Comment:
nit: remove blank space

Reply:
It's a generated file, i think this bring it back to it's original content before my changes last month